src/main/resources/scribe/nutanix-instances.scribe

```diff
@@ -0,0 +1,106 @@
+resource "instance-type" "nutanix" {
+  name                 = "Nutanix Prism Element"
+  code                 = "nutanix"
+  description          = "Spin up any VM Image on your Nutanix Prism Element infrastructure."
+  category             = "cloud"
+  active               = true
+  enabled              = true
+  hasConfig            = false
+  hasSettings          = false
+  hasDeployment        = false
+  hasAutoScale         = true
+  provisionTypeDefault = true
+  optionTypes = [
+    { code = "instanceType.exposePorts" },
+    { code = "provisionType.nutanix.imageId" }
+  ]
+  versions = [
+    "1.0"
+  ]
+  environmentPrefix = "NUTANIX"
+  backupType        = "nutanixSnapshot"
+  stack-tier        = 30
+  viewSet           = "nutanix"
+
+  pluginIconPath          = "nutanix.svg"
+  pluginIconHidpiPath     = "nutanix.svg"
+  pluginIconDarkPath      = "nutanix-dark.svg"
+  pluginIconDarkHidpiPath = "nutanix-dark.svg"
+}
+
+resource "workload-type" "nutanix-1_0" {
+  code             = "nutanix-1.0",
+  shortName        = "nutanix"
+  name             = "Nutanix"
+  ports = []
+  containerVersion = "1.0",
+  repositoryImage  = "",
+  serverType       = "vm",
+  entryPoint       = "",
+  category         = "nutanix",
+  statTypeCode     = "vm",
+  logTypeCode      = "nutanix",
+  checkTypeCode    = "containerCheck",
+  mountPoint       = "/data",
+  actions = [
+    { code = "generic-remove-node" }
+  ],
+  provisionType = "nutanix",
+  backupType    = "nutanixSnapshot"
+}
+
+resource "workload-type-set" "nutanix-1_0-set" {
+  code           = "nutanix-1.0-set",
+  containerType  = workload-type.nutanix-1_0,
+  priorityOrder  = 0,
+  dynamicCount   = true,
+  containerCount = 1
+}
+
+resource "instance-type-layout" "nutanix-1_0-single" {
+  code            = "nutanix-1.0-single"
+  name            = "Nutanix Element VM"
+  sortOrder       = 0
+  instanceVersion = "1.0"
+  description     = "This will provision a single vm"
+  instanceType    = instance-type.nutanix
+  serverCount     = 1
+  portCount       = 0
+  enabled         = true
+  creatable       = true
+  workloads = [
+    { code = "nutanix-1.0-set" }
```

Comment:
Referencing a resource defined by the plugin can be `<resource-type>.<resource_name`, example:

```
workloads = [
    workload-type-set.nutanix-1_0-set
]
```
this will ensure proper dependency ordering and prevent an additional lookup by code.